### ISSUE TITLE
ImportError: No module named pyocd.tools.flash_tool

### ISSUE THREAD

**Issue by kankungyip** [mbedmicro/pyOCD#444]
```bash
$ pip show pyocd
Name: pyocd
Version: 0.13.0
....
$ pyocd-flashtool
Traceback (most recent call last):
  File "/usr/local/bin/pyocd-flashtool", line 7, in <module>
    from pyocd.tools.flash_tool import main
ImportError: No module named pyocd.tools.flash_tool
```

**Comment by flit** [comment]
Thanks for the bug report!

Did you have pyOCD already installed? What OS?

I'm guessing this is a very unfortunate side effect of the (necessary) package name change to all lowercase `pyocd`. On case-insensitive filesystems, pip may not be renaming the previous `pyOCD` to `pyocd` when it updates to version 0.13.0. But Python does perform case-sensitive package imports.

Can you try uninstalling and reinstalling pyOCD to see if that recreates the package directory with the correct name?

If that solves the problem, I'll add some info about this to the top of the README.

**Comment by byq77** [comment]
I had a very similar problem, but on windows. I was getting following exception:
```
E:\mbed_projects\Nucleo_STF401RE\encoder-test\BUILD\NUCLEO_F401RE\GCC_ARM-DEBUG>pyocd-gdbserver -t NUCLEO_F401RE
Traceback (most recent call last):
  File "C:\Python27\Scripts\pyocd-gdbserver-script.py", line 11, in <module>
    load_entry_point('pyocd==0.13.0', 'console_scripts', 'pyocd-gdbserver')()
  File "c:\python27\lib\site-packages\pkg_resources\__init__.py", line 479, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "c:\python27\lib\site-packages\pkg_resources\__init__.py", line 2703, in load_entry_point
    return ep.load()
  File "c:\python27\lib\site-packages\pkg_resources\__init__.py", line 2321, in load
    return self.resolve()
  File "c:\python27\lib\site-packages\pkg_resources\__init__.py", line 2327, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named pyocd.tools.gdb_server
```
I tired to reinstall pyocd with pip, but it didn't help.

However, changing folder name in `C:\Python27\Lib\site-packages\` from `pyOCD` to `pyocd` solved the problem ;-) Now everything is working fine. Maybe it will help somebody.

**Comment by flit** [comment]
I'm able to reproduce the problem with upgrading on Windows.

Although, strangely, I get a slightly different exception:
```
Traceback (most recent call last):
  File "C:\Program Files\Python37\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files\Python37\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "Z:\creed\foo\winenv3\Scripts\pyocd-gdbserver.exe\__main__.py", line 5, in <module>
ModuleNotFoundError: No module named 'pyocd'```

Either way, this seems like a setuptools bug to me. I'll double check, create a setuptools issue and maybe patch, and update the pyOCD README.md with help for doing the upgrade.

**Comment by flit** [comment]
I think I found the problem: I use a separate, clean working copy of the pyOCD repo to perform releases to PyPI. Because I use macOS with a case-insensitive APFS filesystem, when I pulled the 0.13.0 changes into this working copy, git did not rename the `pyOCD` directory to `pyocd`. Nor did setuptools have any issues building the sdist. So the tarball released to PyPI has the incorrect mixed case name for the pyOCD package. This causes different failures depending on whether the end user's filesystem is case-sensitive. Will fix and release 0.13.1 very shortly.
